src/main/java/com/github/bfh/study/slb/provider/eu/entities/PassportInfo.java

```diff
@@ -0,0 +1,23 @@
+package com.github.bfh.study.slb.provider.eu.entities;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlElement;
+
+@XmlAccessorType(XmlAccessType.FIELD)
+public class PassportInfo extends SubEntry {
```

Comment:
@samsamann, there is at the moment no use for this class... Where should this class be used?